lms/djangoapps/instructor_task/tasks.py

```diff
@@ -267,6 +268,22 @@ def generate_certificates(entry_id, xmodule_instance_args):
     return run_main_task(entry_id, task_fn, action_name)
 
 
+@task(base=BaseInstructorTask, routing_key=settings.GRADES_DOWNLOAD_ROUTING_KEY)  # pylint: disable=not-callable
+def generate_specific_students_certificates(entry_id, xmodule_instance_args):  # pylint: disable=invalid-name
```

Comment:
too much duplicate code here. Instead of write a separate task for specific student should modify `generate_students_certificates ` task and pass in a list of those students whose certificates needs to generated.